rebuild.ps1

```diff
@@ -1,2 +1,2 @@
 .\clean.ps1
-.\build.ps1 quick
```

Comment:
You can't rename that - took me 3 months to teach @brockallen the quick.

He actually preferred to call it djquick

https://www.youtube.com/watch?v=J626Qcd7Glo

Reply:
But this is hidden inside `rebuild.ps1`. Do you also call it explicitly with `build.ps1 quick`? If you prefer, I can reinstate `quick`, simply pointing to `copy-pack-output` as a dependency.

Reply:
yes please ;)

Reply:
I pushed a fixup! commit to do that.